### PR TITLE
Enhancement 27 cleanup and refactoring

### LINKED ISSUE #27: Refactoring
Code cleanup and refactoring with Rome

### PR DESCRIPTION
**Pull request by EVP1989**
Resolves #27

### CODE REVIEW

File: src/atoms/error.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
 import { atom } from "jotai";
 
-export const errorAtom = atom<string | undefined>(undefined);
\ No newline at end of file
+export const errorAtom = atom<string | undefined>(undefined);
```
Comment (villejuutila) [inline src/atoms/error.ts:3]:
```suggestion
export const errorAtom = atom<string>();
```
Yields the same typing as `<string | undefined>`

File: src/components/editor/editor.tsx [hunk 2]
```diff
@@ -74,34 +79,34 @@ const PreviewContainer = styled(Box, {
...
-  const [ showAddPage, setShowAddPage ] = useState(false);
+  const [showAddPage, setShowAddPage] = useState(false);
   const setError = useSetAtom(errorAtom);
-  const [ surveyPages, setSurveyPages ] = useAtom(pagesAtom);
-  const [ pageLayouts, setPageLayouts ] = useAtom(layoutsAtom);
-  const [ selectedPage, setSelectedPage ] = useState<number>();
-  const [ isLoading, setIsLoading ] = useState(false);
+  const [surveyPages, setSurveyPages] = useAtom(pagesAtom);
+  const [pageLayouts, setPageLayouts] = useAtom(layoutsAtom);
+  const [selectedPage, setSelectedPage] = useState<number>();
```
Comment (villejuutila) [inline src/components/editor/editor.tsx:86]:
```suggestion
  const [selectedPageNumber, setSelectedPageNumber] = useState<number>();
```
Would probably make more sense when reading the code.

File: src/components/editor/page-properties.tsx [hunk 3]
```diff
@@ -110,53 +123,55 @@ const PageProperties = ({ pageNumber, surveyId }: Props) => {
...
-  
 
   /**
    * Adds new question option to question options
    */
   const addNewQuestionOption = () => {
-    options.length
-      ? setOptions([...options, ""])
-      : setOptions([""]);
+    options.length ? setOptions([...options, ""]) : setOptions([""]);
```
Comment (villejuutila) [inline src/components/editor/page-properties.tsx:174]: Is this overly complicated? Wouldn't spreading the empty `options` array yield same result?

File: src/components/screens/screens-screen.tsx [hunk 4]
```diff
@@ -1,12 +1,10 @@
...
+import strings from "../../localization/strings";
 
 /**
  * Renders screens screen
  */
 const ScreensScreen = () => {
-  return (
-    <div>{strings.generic.notImplemented}</div>
-  )
+  return <div>{strings.generic.notImplemented}</div>;
```
Comment (villejuutila) [inline src/components/screens/screens-screen.tsx:7]: Should this be renamed to `devices-screen.tsx` to maintain consistency of terms across the whole project?

**Review by villejuutila — APPROVED** [review]

File: src/components/screens/screens-screen.tsx (hunk 4 above)
Comment (EVP1989) [reply src/components/screens/screens-screen.tsx:7]: Fixed

File: src/components/editor/page-properties.tsx (hunk 3 above)
Comment (EVP1989) [reply src/components/editor/page-properties.tsx]: Fixed

**Pull request merged**